doc/guides/bluetooth/autopts/autopts-linux.rst

```diff
@@ -1,13 +1,24 @@
 .. _autopts-linux:
 
-AutoPTS on Linux with nRF52 board
-##################################
+AutoPTS on Linux
+################
 
 Overview
 ========
 
 This tutorial shows how to setup AutoPTS client on Linux with AutoPTS server running on Windows 10
-virtual machine. Tutorial covers only nrf52840dk. Tested with Ubuntu 20.4 and Linux Mint 20.4.
+virtual machine. Tested with Ubuntu 20.4 and Linux Mint 20.4.
+
+There are three ways to test zephyr bluetooth host:
```

Comment:
Writing three here means there is one more thing to update if you add another bullet point.
```suggestion
Supported methods to test zephyr bluetooth host:
```